core-libs/setup/ssr/optimized-engine/rendering-strategy-resolver-options.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export interface RenderingStrategyResolverOptions {
+  excludedUrls?: string[];
+  excludedParams?: string[];
+}
+
+export const defaultRenderingStrategyResolverOptions: RenderingStrategyResolverOptions =
+  {
+    excludedUrls: ['checkout', 'my-account'],
+    excludedParams: ['asm'],
+  };
```

Comment:
`should:` as we discussed, this should be a part of the public API.

Reply:
added it to the public api